test/e2e/Apps/BasicPython/extensions.csproj

```diff
@@ -11,7 +11,7 @@
   <ItemGroup>
     <!-- This reference will be dynamically updated by build-e2e-tests.ps1 as part of test setup. -->
     <!-- Do not commit changes to this line unless necessary -->
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="3.3.0" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="3.5.0" />
```

Comment:
Not related to this PR, but should these be project references instead of package references so that we don't have to remember to update them? @andystaples

Reply:
I can open a separate PR for this, or @andystaples if you’d prefer. For now, it’s auto-generated by running build-e2e-test.ps1 locally.